### ISSUE TITLE
Pastes from the command line tool not readable

### ISSUE THREAD

**Issue by carlf** [jeaye/safepaste#3]
I use the command line tool for various stuff usually to send something burn-after-reading. I recently downloaded the script on my laptop and it's producing pastes that are not readable. If I make a new paste in the browser, it works fine. With the command line tool, I get unable to decrypt every time. This is on safepaste.org.

**Comment by jeaye** [comment]
@carlf This is perplexing me, currently; I noticed it about a week ago. I _thought_ it's an issue between versions of OpenSSL, but that doesn't seem to be the case. Just to know, what OS are you on?

I'm seeing it on Arch.

**Comment by jeaye** [comment]
@carlf Oh, one thing to note is that pastes created with the tool are still readable with the tool (this is why I thought it was an incompatible OpenSSL version).

So you can do this just fine:
```bash
$ safepaste < foo | xargs safepaste
```

**Comment by carlf** [comment]
I am indeed on Arch. Mysterious.

```
$ openssl version
OpenSSL 1.1.0f  25 May 2017
```

**Comment by jeaye** [comment]
I've found, even after updating https://github.com/brix/crypto-js to the latest, that it generates different blobs for the same input. This would make sense, with random salts, but those should be configurable (as with OpenSSL, where `-nosalt` produces consistent results for the same input).

Thanks for making an issue for this; I'll leave it open as I make a test case for the crypto-js devs to inspect. Apologies for the inconvenience!

**Comment by jeaye** [comment]
I've pinpointed this down more and made an issue on crypto-js here: https://github.com/brix/crypto-js/issues/101

In short, this does appear to be only affect Arch (and possibly other bleeding edge distro) users. I have verified that ciphertext which fails to decrypt on Arch decrypts perfectly fine on the Gentoo shell account I have. I'll be working with the crypto-js and Arch folks to work out who needs to fix what in order to get this operational again.
